spec/controllers/users/sessions_controller_spec.rb

```diff
@@ -231,6 +232,7 @@
         success: false,
         user_id: user.uuid,
         user_locked_out: false,
+        bad_password_count: 0,
```

Comment:
Could we have a spec demonstrating where the count is greater than 0?

On a similar note, should the count consider the current submission? For example, would it be wrong in this spec that the attempt was unsuccessful (`success: false` 3 lines above) but the `bad_password_count` is `0`? In other words, should the incrementing of the bad password count happen before we log the result, or at least account for it in the logging?

Reply:
Changed the approach.